samples-lib/src/main/java/com/esri/arcgismaps/sample/sampleslib/theme/Color.kt

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2023 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.sampleslib.theme
+
+import androidx.compose.ui.graphics.Color
+
+val colorPrimary = Color(0xFF9243cf)
+
+val md_theme_light_primary = Color(0xFF8434C1)
```

Comment:
I realize some of these colors are modified per our samples requirements. and the others are still from the generated Colors.kt file from the Builder.

I suppose we have updated some of these colors to be same as our current samples theme?

Reply:
I applied the exact color scheme used in the XML color files to the [builder](https://m3.material.io/theme-builder#/custom):
Primary -> `<color name="colorPrimary">#9243cf</color>`
Secondary -> `<color name="colorAccent">#7a2ab7</color>`
Tertiary -> `<color name="colorAccent">#7a2ab7</color>`
Neutral ->   Default builder color

These values are then used to generate a M3 color scheme, exported and then added over to the Color.kt and Theme.kt files. Not much was modified from the generated scheme as the colors matched expected M3 design in the [test sample](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/tree/compose-example/add-graphic-to-map-using-compose). 

I would like to know if the generated theme colors matches expectations for what M3 Kotlin samples will look like moving forward. Or if you would suggest updating them to to match a more custom design scheme?

Reply:
Yes I do think the generated theme colors looks good as is for now. When we start implementing samples using M3, and if we think some color attribute needs an update, we can discuss, made decision and update that time.